bin/bintray/upload

```diff
@@ -114,17 +118,23 @@ bt()
 {
 	cmd="$1"
 	shift
-	# Dummy config to avoid jfrog to stop and ask questions...
-	jfrog bt config --user=nobody --key=nokey --licenses=nolicense
+	if ! $dry_run
+	then
+		# Dummy config to avoid jfrog to stop and ask questions...
+		jfrog bt config --user=nobody --key=nokey --licenses=nolicense
+	fi
 	echo jfrog bt "$cmd" "$@"
-	jfrog bt "$cmd" --user="$user" --key="$key" "$@"
+	if ! $dry_run
+	then
+		jfrog bt "$cmd" --user="$user" --key="$key" "$@"
```

Comment:
Just a minor note: this line appears different from what is `echo`'d above.  Is that intentional?

Reply:
(I recognize the line is unchanged from the previous version, but now that we're actually using an official dry-run mode, having the command accurate might be more important.)